MPCAutofill/cardpicker/templates/cardpicker/patrons.html

```diff
@@ -0,0 +1,50 @@
+{% extends 'cardpicker/base.html' %}
+{% load static %}
+{% block title %}{{ SITE_NAME }} Patreon Supporters{% endblock %}
+{% block content %}
+    <div class="pt-2">
+        <h2>Patreon Supporters</h2>
+        <p>
+            Our site and tools like Proxyshop, Card Crafter, and MTG Art Downloader are supported by the
```

Comment:
can you read the description of the patreon project and include that description here rather than hard-coding this?

Reply:
It can definitely be done, there's a tag line and an "About" section, I felt the wording on Patreon needed to be tailored a little differently to feel natural on the site page, but I'll see if I can rewrite it to work for both!